packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const chalk = require('chalk');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = [];
+    modules.forEach((module) => {
+      if (fs.existsSync(`${filePath}/mapper/${module}/uid-mapping.json`)) {
+        let mappedIds = fs.readFileSync(`${filePath}/mapper/${module}/uid-mapping.json`, 'utf-8');
+        mappedIds = JSON.parse(mappedIds);
+        uidMapping = { ...uidMapping, ...mappedIds };
+      } else {
+        console.log(chalk.red(`Mapper Does not exist for module '${module}'`));
+      }
+    });
+
+    return uidMapping;
+  };
+
+  const getEntries = async (ct) => {
+    try {
+      let entries = [];
+      let skip = 0;
+      let limit = 1;
+      let count = 0;
+      while (skip <= count) {
+        const res = await stackSDKInstance.contentType(ct).entry().query({ include_count: true, skip, limit }).find();
+        count = res.count;
+        skip += limit;
+        entries = [...entries, ...res.items];
+      }
+      return entries;
+    } catch (err) {
+      console.log(err);
+      console.log(chalk.red(`No entry is found for Content-type ${ct}`));
+      return [];
```

Comment:
@cs-raj don't think this catch block has to be returned to. From the location where you are calling this function, you may manage this condition.